source/game.popularity.person.bmx

```diff
@@ -117,7 +117,7 @@ Type TGameModifierPopularity_ModifyPersonPopularity extends TGameModifierPopular
 	Method Init:TGameModifierPopularity_ModifyPersonPopularity(data:TData, extra:TData=null)
 		Super.Init(data, extra)
 
-		If data And data.GetString("time") Then self.data = data.copy()
+		InitTimeDataIfPresent(data)
```

Comment:
Da InitTimeDataIfPresent nun eigentlich ueberall "dabei ist" ... hast Du irgendwie mitbekommen, ob es bei anderen Modifiern schadet, `Super.Init(data, extra)` aufzurufen? Denn dann koennte der TimeInit ja dort landen.

Reply:
Ich habe das InitTime nur bei den Modifiern hinzugepackt, bei denen es für die Datenbank sinnvoll erscheint, nicht bei allen.
Das Aufräumen und Verfeinern würde ich wirklich lieber in Ruhe separat nochmal machen. Auch weil man sich nochmal genau darüber Gedanken machen muss, wie sich verzögerte Mehrfacheffekte verhalten (sollen) - da das time aus data gelöscht wird, wäre die Verzögerung bei einer zweiten Ausführung nicht mehr dabei, was unerwartet sein könnte.

Reply:
OK